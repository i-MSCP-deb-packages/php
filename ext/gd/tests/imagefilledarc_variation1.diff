001+ cc98cc8cc1bc370d0e6e23319b72ab6a
001- b77bbb8207e5adbebfcc8bd1c4074305