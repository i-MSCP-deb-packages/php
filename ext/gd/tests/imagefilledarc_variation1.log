
---- EXPECTED OUTPUT
b77bbb8207e5adbebfcc8bd1c4074305
---- ACTUAL OUTPUT
cc98cc8cc1bc370d0e6e23319b72ab6a
---- FAILED
