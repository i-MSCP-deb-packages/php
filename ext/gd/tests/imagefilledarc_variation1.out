cc98cc8cc1bc370d0e6e23319b72ab6a